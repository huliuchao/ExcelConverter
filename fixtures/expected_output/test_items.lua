-- test_items Data
-- Auto-generated by Excel Converter

return {
    [1001] = {
        ID = 1001,
        Name = "测试剑",
        Attributes = {Attack = 100, Defense = 50}
    },
    [1002] = {
        ID = 1002,
        Name = "测试盾",
        Attributes = {Attack = 10, Defense = 200}
    },
    [1003] = {
        ID = 1003,
        Name = "测试药水",
        Attributes = {Attack = 0, Defense = 0}
    }
} 