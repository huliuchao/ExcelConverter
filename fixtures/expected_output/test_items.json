[
  {
    "ID": 1001,
    "Name": "测试剑",
    "Attributes": {
      "Attack": 100,
      "Defense": 50
    }
  },
  {
    "ID": 1002,
    "Name": "测试盾",
    "Attributes": {
      "Attack": 10,
      "Defense": 200
    }
  },
  {
    "ID": 1003,
    "Name": "测试药水",
    "Attributes": {
      "Attack": 0,
      "Defense": 0
    }
  }
] 